Bottom: 82d055e73b7f44fdbe4a05b3ef99474a1c4eff84
Top:    b104780a3f91a0df48d2744293ef1d673562b8d0
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-27 14:34:35 +0800

Refresh of autofs-5.0.7-dont-prode-rdma-mounts.patch

---

diff --git a/modules/mount_nfs.c b/modules/mount_nfs.c
index e3d7746..5424d74 100644
--- a/modules/mount_nfs.c
+++ b/modules/mount_nfs.c
@@ -125,6 +125,11 @@ int mount_mount(struct autofs_point *ap, const char *root, const char *name, int
 				end--;
 
 			o_len = end - cp + 1;
+
+			if (strncmp("proto=rdma", cp, o_len) == 0 ||
+				   strncmp("rdma", cp, o_len) == 0)
+				rdma = 1;
+
 			if (strncmp("nosymlink", cp, o_len) == 0) {
 				warn(ap->logopt, MODPREFIX
 				     "the \"nosymlink\" option is depricated "
@@ -137,9 +142,6 @@ int mount_mount(struct autofs_point *ap, const char *root, const char *name, int
 				flags &= ~MOUNT_FLAG_USE_WEIGHT_ONLY;
 			} else if (strncmp("use-weight-only", cp, o_len) == 0) {
 				flags |= MOUNT_FLAG_USE_WEIGHT_ONLY;
-			} else if (strncmp("proto=rdma", cp, o_len) == 0 ||
-				   strncmp("rdma", cp, o_len) == 0) {
-				rdma = 1;
 			} else {
 				if (strncmp("vers=4", cp, o_len) == 0 ||
 				    strncmp("nfsvers=4", cp, o_len) == 0)
